php/src/Google/Protobuf/Internal/Message.php

```diff
@@ -826,7 +916,7 @@ private function convertJsonValueToProtoValue(
         }
     }
 
-    private function mergeFromJsonArray($array)
+    private function mergeFromArray($array, $is_json = false)
```

Comment:
Do we still plan to provide mergeFromArray? If yes, I would like to just have $array as parameter. We could have another mergeFromArrayImpl in that case.

Reply:
Good call, I made `mergeFromArray` public and added a private `mergeFromArrayImpl`.